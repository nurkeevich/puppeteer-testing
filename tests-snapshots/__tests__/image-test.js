const puppeteer = require("puppeteer");
const { toMatchImageSnapshot } = require("jest-image-snapshot");

expect.extend({ toMatchImageSnapshot });

describe("Home page snapshot", () => {
  let browser;
  let page;

  beforeAll(async function() {
    browser = await puppeteer.launch({
      headless: true
    });

    page = await browser.newPage();
  });

  afterAll(async function() {
    await browser.close();
  });

  test("snapshot home page", async () => {
      await page.goto("https://example.com");
      const image = page.screenshot();
      expect(image).toMatchSnapshot();
  });
});
