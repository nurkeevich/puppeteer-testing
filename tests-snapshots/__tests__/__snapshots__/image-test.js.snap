// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page snapshot snapshot home page 1`] = `Promise {}`;
